robottelo/performance/graph.py

```diff
@@ -0,0 +1,97 @@
+import pygal
+
+
+def generate_bar_chart_stat(stat_dict, head, filename, lengend):
+    """Generate Bar chart for stat on concurrent subscription"""
+    bar_chart = pygal.Bar()
+    bar_chart.title = head
+    bar_chart.x_labels = ('min', 'mean', 'max', 'std')
+    bar_chart.x_title = 'Statistics'
+    bar_chart.y_title = 'Time (s)'
+    for key in range(len(stat_dict)):
+        bar_chart.add(
+            '{0}-{1}'.format(lengend, key),
+            stat_dict.get(key)
+        )
+    bar_chart.render_to_file(filename)
+
+
+def generate_line_chart_bucketized_stat(stat_dict, head, filename,
+                                        bucket_size, num_buckets):
```

Comment:
Please do not list arguments in a hanging block like this. Ideally, use a minimal indentation level and place each argument on its own line.

```python
def generate_line_chart_bucketized_stat(
        stat_dict,
        head,
        filename,
        bucket_size,
        num_buckets):
    """Generate Line chart for stat on concurrent subscription"""
```